odoo/addons/test_impex/tests/test_load.py

```diff
@@ -153,18 +140,14 @@ def test_exported(self):
         ], values(records))
 
     def test_falses(self):
-        for lang, source, value in [('fr_FR', 'no', u'non'),
-                                    ('de_DE', 'no', u'nein'),
-                                    ('ru_RU', 'no', u'нет'),
-                                    ('nl_BE', 'false', u'vals'),
-                                    ('lt_LT', 'false', u'klaidingas')]:
-            self.add_translations('test_import.py', 'code', lang, (source, value))
+        for lang in ['fr_FR', 'de_DE', 'ru_RU', 'nl_BE', 'lt_LT']:
+            self.env['res.lang']._activate_lang(lang)
         falses = [[u'0'], [u'no'], [u'false'], [u'FALSE'], [u''],
-                  [u'non'], # no, fr
-                  [u'nein'], # no, de
-                  [u'нет'], # no, ru
-                  [u'vals'], # false, nl
-                  [u'klaidingas'], # false, lt,
+                  [u'faux'], # no, fr
+                  [u'falsch'], # no, de
```

Comment:
"falsch" is "false" in german, "nein" is "no" in german, either the comment or the content needs fixing